Adamant/Modules/Chat/View/Managers/ChatDialogManager.swift

```diff
@@ -201,13 +201,15 @@ private extension ChatDialogManager {
         alert.addAction(
             .init(
                 title: .adamant.chat.reviewNodesList,
-                style: .default,
+                style: .destructive,
                 handler: { [weak self] _ in
                     self?.viewModel.didTapAdmNodesList.send(())
                 }
             )
         )
-        alert.addAction(.init(title: .adamant.alert.cancel, style: .cancel))
+        let cancelButton = UIAlertAction(title: .adamant.alert.cancel, style: .default)
+        cancelButton.setValue(UIColor.blue, forKey: "titleTextColor")
```

Comment:
Hey;

Really unsure about setting title text color this way. It does not look as good as when using default tint colour; also it does not change the title text colour on macOS version for this button.

Reply:
As discussed we are good to not set colour for now, rather investigate how to do it properly using UIAppearance proxy instead.